### PR TITLE
Privacy modifications

### PR DESCRIPTION
**Pull request by sarahcec**
Modified and clarified some privacy language that was added earlier.

### CODE REVIEW

**Comment by paul-grassi** [comment]
@nbl-nist check it out before I accept this pull request.

File: sp800-63c/sec4_federation.md [hunk 1]
```diff
@@ -52,20 +51,20 @@ In this model, a third-party sits in the middle of the transaction and communica
...
+Brokers can enable simplified technical integrations between the RP and CSP by eliminating the need for multiple point to point integrations, which can be onerous for protocols which do not support [dynamic registration](#dynamic-registration). If implemented in very specific ways, brokers can also provide some business confidentiality and transfer some of the privacy risks of point to point federation described above by passing the assertions while blinding the participants on either side of the transaction to each other. For example, organizations may not wish to reveal their subscriber lists to each other and some implementations of blinding technology somewhat prevent the capability for CSPs or RPs to track and profile subscribers. However, this model transfers this capability to the broker.  While the use of brokers can give the appearance of complete privacy preservation, in many instances the implementation of broker technology still allows the tracking of subscribers, CSPs, and RPs in different ways.
```
Comment (nbl-nist) [inline sp800-63c/sec4_federation.md]: For example sentence - remove somewhat, add within the authentication transaction.

File: sp800-63c/sec4_federation.md [hunk 2]
```diff
@@ -52,20 +51,20 @@ In this model, a third-party sits in the middle of the transaction and communica
...
 
-There is a spectrum of blinding technologies that offer varying levels of privacy protection. However, as the level of blinding increases, so does the technical implementation complexity. Although privacy policies may dictate appropriate use by the CSP, RP, and the broker, blinding technology is far more effective, by making the data unavailable.
+While some broker deployments offer no additional privacy protection, some can offer limited additional levels of privacy to the subscriber through a variety of blinding technologies.  However, as the level of blinding increases, so does the technical implementation complexity. Privacy policies may dictate appropriate use by the CSP, RP, and the broker,  but blinding technology can increase effectiveness of these policies by making the data more difficult to access.
```
Comment (nbl-nist) [inline sp800-63c/sec4_federation.md]: While some broker deployments offer no additional privacy protection to the subscriber, others may  through a variety of blinding technologies. 

Add to however sentence "technical and operational implementation complexity."

Add at end, the following list illustrates a spectrum of blinding implementations.

File: sp800-63c/sec5_strength.md [hunk 3]
```diff
@@ -62,7 +62,11 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
 
 #### 5.2.5. Pseudonymous Identifiers
 
-To prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier, pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP. The CSP SHALL generate a different identifier for each RP.
+In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. If the subscriber has not consented to use of a common identifier, pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP.
```
Comment (nbl-nist) [inline sp800-63c/sec5_strength.md]: Eliminate consent.

File: sp800-63c/sec5_strength.md [hunk 4]
```diff
@@ -62,7 +62,11 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
...
+
+When unique pseudonymous identifiers are used with RPs along side of identity attribute bundles, it is often possible for multiple colluding RPs to fully identify and correlate a subscriber across systems using these attributes. Even though a different pseudonymous identifier was used at each RP, the RPs are still able to compare other identifying attributes such as names, email addresses, physical addresses, and phone numbers, among other common attributes. In such cases, the privacy risk is similar to the subscriber not having used a pseudonymous identifier at all.
```
Comment (nbl-nist) [inline sp800-63c/sec5_strength.md]: fold into privacy considerations

File: sp800-63c/sec6_presentation.md [hunk 5]
```diff
@@ -4,9 +4,7 @@
...
-
-The manner of presentation may impact who is in the best position to provide notice and obtain confirmation from the subscriber. Accordingly, the CSP, RP, and any broker SHALL agree in advance on who will provide notice and obtain confirmation.
+Positive confirmation SHALL be obtained from the subscriber before any attributes about the subscriber are transmitted to any RP. If the protocol in use allows for optional attributes, the subscriber SHALL be given the option to decide whether transmit those attributes to the RP. These requirements do not apply to cases where the subscriber has already consented to the attribute release or has a reasonable expectation of attribute sharing, such as in cases where the RP and the CSP are the same legal entity.
```
Comment (nbl-nist) [inline sp800-63c/sec6_presentation.md]: Delete These requirements do not apply sentence.

File: sp800-63c/sec5_strength.md [hunk 6]
```diff
@@ -62,7 +62,11 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
 
 #### 5.2.5. Pseudonymous Identifiers
```
Comment (jricher) [inline sp800-63c/sec5_strength.md]: "Pairwise" or "Directed" instead of Pseudonymous?

File: sp800-63c/sec5_strength.md (hunk 3 above)
Comment (jricher) [inline sp800-63c/sec5_strength.md]: Add section on identifier generation requirements.

File: sp800-63c/sec4_federation.md [hunk 7]
```diff
@@ -13,8 +13,9 @@ The subscriber authenticates to the CSP using some form of primary credential, a
...
 
-The CSP SHALL not disclose information on subscriber activities with an RP to any party, nor use the 
-information for any purpose other than federated authentication or to comply with law or legal process. The CSP SHOULD employ technical measures to provide unlinkability and prevent subscriber activity tracking and profiling.
+The CSP SHALL NOT disclose information on subscriber activities with an RP to any party, nor use the information for any purpose other than federated authentication or to comply with law or legal process except in the case of a specific user request. The CSP SHOULD employ technical measures to provide unlinkability and prevent subscriber activity tracking and profiling.
+
+Subscriber activity may be disclosed for security purposes such as communication of compromised subscriber accounts.
```
Comment (jricher) [inline sp800-63c/sec4_federation.md]: I think this MAY needs to be normative.

File: sp800-63c/sec4_federation.md [hunk 8]
```diff
@@ -52,20 +53,24 @@ In this model, a third-party sits in the middle of the transaction and communica
...
+Brokers can enable simplified technical integrations between the RP and CSP by eliminating the need for multiple point to point integrations, which can be onerous for protocols which do not support [dynamic registration](#dynamic-registration). If implemented in very specific ways, brokers can also provide some business confidentiality and transfer some of the privacy risks of point to point federation described above by passing the assertions while blinding the participants on either side of the transaction to each other. 
+
+For example, organizations may not wish to reveal their subscriber lists to each other. Some implementations of blinding technology can prevent the capability for CSPs or RPs to track and profile subscribers within the context of a specific authentication transaction. However, the broker model transfers this capability to the broker. Additionally, CSPs and RPs may still be able to track and profile subscribers through activity monitoring.
```
Comment (jricher) [inline sp800-63c/sec4_federation.md]: Awkward wording, suggest replacing second and third sentence with:

> Some implementations of blinding technology can prevent CSPs or RPs from tracking and profiling subscribers within the context of an authentication transaction. However, the broker model transfers this tracking and profiling capability to the broker itself.

**Pull request merged**